web-server/src/controllers/auth/changeGeneral.ts

```diff
@@ -0,0 +1,123 @@
+import { Request, Response, NextFunction } from 'express';
+import validator from 'validator';
+
+import { AppDataSource } from 'orm/data-source';
+import { User } from 'orm/entities/users/User';
+import { CustomError } from 'utils/response/custom-error/CustomError';
+
+import { Countries } from '../../orm/entities/users/types';
+
+export const changeGeneral = async (req: Request, res: Response, next: NextFunction) => {
+  const {
+    id,
+    username,
+    password,
+    newEmail,
+    newUsername,
+    newPassword,
+    newPasswordConfirm,
+    // newIsAdmin,
+    newSchool,
+    newFirstName,
+    newLastName,
+    newCountry,
+  } = req.body;
```

Comment:
Kinda strange to get (old) `username` and `password` here. Also, why is everything `newX` rather than just `X`? :D